container_pipeline/workers/base.py

```diff
@@ -29,16 +29,17 @@ def notify(self, data):
         """
         self.queue.put(json.dumps(data), 'master_tube')
 
-    def export_logs(self, logs, destination):
+    def export_logs(self, logs, destination, json_logs=False):
```

Comment:
I think it will be better if the caller can call this method ``JSON``ified logs.